patterns/atoms/base/_vendor.scss

```diff
@@ -1,4 +1,49 @@
-// Reset the default styles
-@import 'node_modules/reset-css/_reset.scss';
-// Wordpress Default Styles
-@import 'node_modules/wp.scss/wp';
+// Required
+@import "~bootstrap/scss/functions";
+@import "~bootstrap/scss/variables";
+@import "~bootstrap/scss/mixins";
+
+// Optional
+@import "~bootstrap/scss/root";
+@import "~bootstrap/scss/reboot";
+@import "~bootstrap/scss/type";
+@import "~bootstrap/scss/code";
+@import "~bootstrap/scss/grid";
+@import "~bootstrap/scss/forms";
+@import "~bootstrap/scss/buttons";
+@import "~bootstrap/scss/nav";
+@import "~bootstrap/scss/navbar";
+@import "~bootstrap/scss/carousel";
+@import "~bootstrap/scss/transitions";
+@import "~bootstrap/scss/utilities";
+
+/**
```

Comment:
this animations can be removed, what do you think?

Reply:
transitions is necessary for the hamburger menu and utilities for the carousel